.github/actions/action-npm-release/action.yml

```
@@ -0,0 +1,72 @@
+     default: true
+     type: boolean
+     description: skip deployment to npm registry
+  npm-token: 
+   type: string
+   required: true
+   description: npm token for release
+     
+outputs:
+  ARTIFACT_NAME:
+    description: "artifact name from job artifact"
+    value:  ${{ steps.node.outputs.artifact-name }}
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)